### PR TITLE
chore: debug

### PR DESCRIPTION
**Pull request by Keyrxng**
something aint right with config

Resolves #

Quality Assurance: 
- https://github.com/Keyrxng/ubiquibot/issues/4#issuecomment-1732367813

### CODE REVIEW

**Comment by Keyrxng** [comment]
@pavlovcik @0xcodercrane

**Comment by Keyrxng** [comment]
- https://github.com/ubiquity/ubiquibot/issues/812#issuecomment-1732368338

Just ran /start and got a max active on this repo, so idk what it is I've got wrong here as I've pulled fresh from development to debug /ask.

**Comment by Keyrxng** [comment]
idk where I'm going wrong.

I do know that:
- I have no repo config or org config

But the bot should be able to run on it's own default config if those other two fail surely?

**Comment by pavlovcik** [comment]
> But the bot should be able to run on it's own default config if those other two fail surely?

It's supposed to be able to do that yes

**Comment by Keyrxng** [comment]
- https://github.com/Keyrxng/UbiquityBotTesting/issues/58#issuecomment-1732373425

``/help``
```js
export const generateHelpMenu = () => {
  let helpMenu = "### Available commands\n```";
  const commands = userCommands();
  commands.map((command) => {
```

/ask not being displayed in the help table despite looping through all commands despite being in ``commands``:

> "  ASK = "/ask", // ask GPT a question"

**Comment by pavlovcik** [comment]
Maybe your code is out of date. Pull upstream

**Comment by Keyrxng** [comment]
I'm up to date otherwise there would be a notification on this pr to update to current commit and my branch 100% up to date just pulled earlier and double checked right there... wtf

**Comment by Keyrxng** [comment]
Not sure what to do here.

I can't refork the repo I'd need to delete first and lose my current work.

I'm 100% up to date with the current dev branch.

I have to figure out setting up an organization to get around the default config issue atm to be able to debug the actual bot.

If your help command is showing ask, and your getting rate limited with tokens then it's registering as a command.

I noticed during my trials that if I spammed an /ask on two repos before the first had finalized one or both was likely to fail.

So it may be a good issue to have raised to create a queue for gpt requests (I'd say to the same api key as that's probs the blocker but likely there will only ever be one api in use for folks)

File: src/configs/ubiquibot-config-default.ts
Comment (pavlovcik) [inline src/configs/ubiquibot-config-default.ts:1]: These need to be off by default

**Review by 0x4007 — CHANGES_REQUESTED** [review]

**Comment by pavlovcik** [comment]
I couldn't find logs related to it that's why I filed the issue but I could add you to the Supabase project if you dm me your Supabase account email

**Pull request closed without merging**